src/main/java/org/folio/fqm/service/EntityTypeFlatteningService.java

```diff
@@ -76,6 +77,11 @@ private EntityType getFlattenedEntityType(UUID entityTypeId, EntityTypeSource so
         UUID sourceEntityTypeId = UUID.fromString(source.getId());
         // Recursively flatten the source and add it to the flattened entity type
         EntityType flattenedSourceDefinition = getFlattenedEntityType(sourceEntityTypeId, source, tenantId);
+        // If the original entity type already supports cross-tenant queries, we can skip this. Otherwise, copy the nested source's setting
+        // This effectively means that if any nested source supports cross-tenant queries, the flattened entity type will too
+        if (!Boolean.TRUE.equals(flattenedEntityType.getCrossTenantQueriesEnabled())) {
+          flattenedEntityType.crossTenantQueriesEnabled(flattenedSourceDefinition.getCrossTenantQueriesEnabled());
+        }
```

Comment:
This logic is kind of roundabout. It's not _awful_ but I do have to mentally run through every scenario for it to really make sense.

1. If `flattenedEntityType` is ECS-enabled, then it will just stay enabled.
2. If it isn't enabled and the nested source also isn't, it stays disabled.
3. If it doesn't have it enabled and the nested source _does_, then this will make `flattenedEntityType` ECS-enabled.
4. If it then runs into a nested source where it's disabled, it will automatically get skipped because `flattenedEntityType` is already ECS-enabled, so it falls into scenario 1